.stylelintrc.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
     rules: {
         "block-no-empty": true,
         "color-hex-case": "lower",
-        "max-nesting-depth": 3,
+        "max-nesting-depth": 4,
```

Comment:
We did discuss it and Caroline argued that our internal style guide suggested a limit of 4 and not 3. Seeing as the exception is being introduced everywhere anyway, I think it's appropriate to adjust the limit to 4. But if you have counter arguments, do not hesitate.